src/http.cc

```diff
@@ -2232,25 +2255,7 @@ HttpStateData::sendRequest()
                                     Dialer, this,  HttpStateData::wroteLast);
     }
 
-    /*
-     * Is keep-alive okay for all request methods?
```

Comment:
This questions seems valid/interesting. We should either keep it or give an explicit answer in the PR description.